openverse_catalog/dags/common/loader/s3.py

```diff
@@ -28,6 +38,34 @@ def copy_file_to_s3_staging(
     s3.load_file(tsv_file_path, staging_key, bucket_name=s3_bucket)
 
 
+def copy_file_to_s3(
+    tsv_file_path,
+    s3_bucket,
+    s3_prefix,
+    aws_conn_id,
+    ti,
+):
+    """
+    Copy a TSV file to S3 with the given prefix.
+    The TSV's version is pushed to the `tsv_version` XCom, and the constructed
+    S3 key is pushed to the `s3_key` XCom.
+    The TSV is removed after the upload is complete.
+    """
+    if tsv_file_path is None:
+        raise FileNotFoundError(f"Expected file {tsv_file_path} was not provided")
```

Comment:
In this case it's not so much that the file wasn't provided but the path itself, so there shouldn't be a `tsv_file_path` to log:
```suggestion
        raise FileNotFoundError(f"No TSV file path was provided.")
```

Reply:
:woman_facepalming: totally true, I'll update this!